pages/know-your-rights/index.tsx

```diff
@@ -0,0 +1,177 @@
+  }),
+)
+const KnowYourRights = () => {
+  const classes = useStyles()
+  const [open, setOpen] = React.useState(false)
+  const [activeItem, setActiveItem] = React.useState({})
+  console.log('styles')
+  const flyers = [
+    {
+      name: 'Police Interaction for Black and Brown People',
+      organization: 'ACLU',
+      profilePic: 'https://i.ibb.co/xFcqtKz/KYR-Black-Brown-Police-ENGLISH.jpg',
+    },
+    {
+      name:
+        'What to do if questioned by Police, FBI, Custom Agents or Immigration Officers',
+      organization: 'ACLU',
+      profilePic:
+        'https://i.ibb.co/VDHYQQd/KYR-Questioned-by-Police-ENGLISH.jpg',
+    },
+    {
+      name: 'Know your rights: Police Interactions ',
+      organization: 'ACLU',
+      profilePic:
+        'https://i.ibb.co/gP9yVBJ/KYR-Police-Interactions-English.jpg',
+    },
+    {
+      name: 'Know your rights: the TRUST ACT ',
+      organization: 'ACLU',
+      profilePic: 'https://i.ibb.co/vL9bChn/TRUST-ACT-KYR.jpg',
+    },
+    {
+      name: 'Know your rights on trains and buses ',
+      organization: 'ACLU',
+      profilePic:
+        'https://i.ibb.co/XxKGDYD/KYR-Trains-and-Buses-Immigration-ENGLISH.jpg',
+    },
+  ]
+  const handleOpen = item => {
+    setActiveItem(item)
+    setOpen(true)
+  }
+  const handleClose = () => {
+    setOpen(false)
+  }
+  return (
+    <>
+      <Head>
+        <title>{`Santa Barbara Reentry | Know Your Rights`}</title>
+      </Head>
+      <div>
+        <Typography
+          style={{ marginTop: '3rem' }}
+          align="center"
+          gutterBottom
+          variant="h2"
+          component="h3"
+        >
+          Know your Rights
+        </Typography>
+        <Grid
+          container
+          spacing={2}
+          direction="row"
+          justify="flex-start"
+          alignItems="flex-start"
+        >
+          {flyers.map((flyer, key) => {
+            return (
+              <Grid item xs={12} sm={3} key={key}>
+                <Card className={classes.root}>
+                  <CardActionArea>
+                    <button type="button" onClick={() => handleOpen(flyer)}>
+                      <CardMedia
+                        className={classes.media}
+                        image={flyer.profilePic}
+                        title="Contemplative Reptile"
+                      />
+
+                      <CardContent className={classes.cardContent}>
+                        <Typography gutterBottom variant="h5" component="h2">
+                          {flyer.name}
+                        </Typography>
+
+                        <Typography gutterBottom variant="h5" component="h2">
+                          {flyer.organization}
+                        </Typography>
+                      </CardContent>
+                    </button>
+                  </CardActionArea>
+                </Card>
+              </Grid>
+            )
+          })}
+          <Grid item xs={12} sm={6}>
+            <Modal
```

Comment:
Modals not rendering documents properly
![Screen Shot 2021-03-02 at 10 17 22 AM](https://user-images.githubusercontent.com/72828520/109695485-cbaa6f80-7b40-11eb-8cad-a1a99d7aace4.png)
![Screen Shot 2021-03-02 at 10 17 12 AM](https://user-images.githubusercontent.com/72828520/109695489-ce0cc980-7b40-11eb-9057-88213c4f2b6f.png)